doc/reference.conf

```diff
@@ -457,8 +457,10 @@ privset "local_op" {
 	 * oper:resv:            allows /quote resv/unresv
 	 * oper:cmodes:          allows cmode +LP
 	 * oper:operwall:        allows the oper to send/receive operwalls
-	 * oper:spy:             allows 'operspy' features to see through +s
+	 * oper:spy_channel:     allows 'operspy' features to see through +s
 	 *                       channels etc. see /quote help operspy
+	 * oper:spy_user:        allows 'operspy' features to see user connection
+	 *			 details. see /quote help operspy
```

Comment:
This is continued with tabs, the others are using spaces.

Reply:
```suggestion
	 *                       details. see /quote help operspy
```